src/extension.ts

```diff
@@ -193,10 +193,17 @@ async function patch({ htmlFile, jsFile, bypassMessage }: PatchArgs) {
  * This function checks the current 'window.controlsStyle' configuration setting. If it's set to 'native',
  * it displays an information message to the user and automatically updates the setting to 'custom'
  * in the global configuration.
+ * 
+ * Note: This setting only exists on Windows. On other platforms, this function returns false immediately.
  *
- * @returns `true` if the controls style was updated, `false` if it was already set to 'custom'.
+ * @returns `true` if the controls style was updated, `false` if it was already set to 'custom' or on non-Windows platforms.
```

Comment:
The statement about false is incorrect. It should be `or on darwin platform.`

```suggestion
 * @returns `true` if the controls style was updated, `false` if it was already set to 'custom' or or on darwin platform.
```